src/applications/personalization/dashboard/actions/claims.js

```diff
@@ -42,11 +42,49 @@ export function fetchAppealsSuccess(response) {
   };
 }
 
+const recordAppealsAPIEvent = ({
+  startTime,
+  success,
+  error,
+  apiName = 'GET appeals /v0/appeals',
+}) => {
+  const event = {
+    event: 'api_call',
+    'api-name': apiName,
+    'api-status': success ? 'successful' : 'failed',
+  };
+  if (error) {
+    event['error-key'] = error;
+  }
+  if (startTime) {
+    const apiLatencyMs = roundToNearest({
+      interval: 5000,
+      value: Date.now() - startTime,
+    });
+    event['api-latency-ms'] = apiLatencyMs;
+  }
+  recordEvent(event);
+  if (event['error-key']) {
+    recordEvent({
+      'error-key': undefined,
+    });
+  }
```

Comment:
This looks like it is triggering 2 record events calls instead of one. Is that how it is supposed to be?

Reply:
thanks for catching this!! oof seems like this is sprinkled in a few places through `/applications/`, and this is the only explanation i found:

https://github.com/department-of-veterans-affairs/vets-website/blob/c04ad21debdff76287f0419d359353fd676aee85/src/applications/personalization/profile/ducks/communicationPreferences.js#L44-L49 

i'm going to leave it out for now to see if it changes anything in the GA dashboard. 

-- 

## to test new change:
1. go to `/dashboard/mocks/server.js` and change line 35 to

```
  'GET /v0/appeals': (_req, res) => {
    return res.status(502).json({
      errors: [
        {
          title: 'Server error',
          status: '502',
        },
      ],
    });
  },
```

2. check console for the following event:

![Screenshot 2024-03-19 at 2 04 45 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/8542413/ccce1441-2d89-4d89-9f21-4f140eb40a6e)